packages/docs/package.json

```diff
@@ -17,13 +17,16 @@
     "@material-ui/core": "^4.11.2",
     "@mdx-js/react": "^1.6.21",
     "@techboi/consent-manager": "*",
+    "@techboi/consent-manager-integration-youtube": "*",
     "clsx": "^1.1.1",
     "final-form": "^4.20.1",
     "react": "^16.8.4",
     "react-dom": "^16.8.4",
     "react-final-form": "^6.5.2",
     "react-youtube": "^7.13.0",
-    "use-persisted-state": "^0.3.3"
+    "simple-icons": "^4.7.0",
+    "use-persisted-state": "^0.3.3",
+    "wcag-contrast": "^3.0.0"
```

Comment:
Can you actually use the consent manager in this form without having this dependency? It's part of the core integration helpers, and we de-structure from it / use it as a hard import.

I think for now this is a dependency rather than a peer-dependency :/

Reply:
both are very, very small.

https://bundlephobia.com/result?p=wcag-contrast@3.0.0

and simple icons supports tree-shake

Reply:
<img width="909" alt="Screenshot 2021-03-12 at 00 54 22" src="https://user-images.githubusercontent.com/1737026/110870814-7eab6380-82cd-11eb-9581-3e6db7a2d3ef.png">

Reply:
I cleaned up the dependencies, the packages now depend on what they actually use :)